### ISSUE TITLE
Verify response objects 

### ISSUE THREAD

**Issue by TimoGlastra** [2060-io/credo-ts-didweb-anoncreds#3]
As it's passed from a remote server we can't be sure the objects are valid. Should we do some sort of object validation before returning it?

**Comment by genaris** [comment]
Yeah I think that would be nice, and also a good way to start using Zod!
